markdown/ch3.markdown

```diff
@@ -70,11 +70,13 @@ inline` IPython magic to make our images appear below the code:
 
 ```python
 %matplotlib inline
+
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
```

Comment:
- I agree. At the moment the comments get more brief, then disappear. I think it should just be a very short comment, that's the same from 2 onwards (in case people pick up the book in the middle).
- Agreed, I tried to do that, but clearly missed one.
- Agreed, own cell. I think they all are?